java/code/src/com/suse/manager/webui/menu/MenuTree.java

```diff
@@ -497,9 +497,13 @@ private MenuItem getHelpNode(String docsLocale) {
     }
 
     private MenuItem getAboutNode(String docsLocale) {
-        return new MenuItem("About Spacewalk").withIcon("fa-question-circle")
-            .addChild(new MenuItem("Overview").withPrimaryUrl("/rhn/help/about.do"))
-            .addChild(new MenuItem("Sign In").withPrimaryUrl("/rhn/manager/login"))
+        return new MenuItem("Help").withIcon("fa-question-circle")
+            .addChild(new MenuItem("API")
```

Comment:
Is this change intentional? The menu item for API has moved upwards in the listing, previously it was at the bottom.

Reply:
Yes, so that when the user clicks on the `API Documentation` link from the login page it will open the API overview page as help has no landing page.

Reply:
Sounds good, I missed that discussion. 👍